debvader/deblender.py

```diff
@@ -97,60 +97,42 @@ def get_residual_field(self, res_deblend=None):
         deblended_image = self.field_image.copy()
 
         if res_deblend is not None:
+
             for isolated_galaxy_row in res_deblend:
 
                 # TODO: may be use some inbuild padding function?
```

Comment:
`np.pad` ?

Reply:
Sorry, this is commented line is now outdated. I just forgot to remove it. (I am no longer using padded images here)